lnbits/core/services.py

```diff
@@ -805,6 +806,7 @@ async def create_user_account(
     password = pwd_context.hash(password) if password else None
 
     account = await create_account(user_id, username, email, password, user_config)
+    await create_wallet(user_id=account.id, wallet_name=wallet_name)
```

Comment:
```suggestion
    wallet = await create_wallet(user_id=account.id, wallet_name=wallet_name)
    assert wallet, "could not create wallet"
```

Reply:
this `assert` is redundant, there is one already in `crud.py/create_wallet`